The grbl main will be called from an STM32F4 specific main.
--- grbl-master/main.c	2014-03-18 21:09:06.000000000 -0700
+++ grbl-master/main.c	2014-03-18 21:09:51.000000000 -0700
@@ -42,13 +42,13 @@
 // Declare system global variable structure
 system_t sys; 
 
-int main(void)
+int grbl_main(void)
 {
   // Initialize system
   serial_init(); // Setup serial baud rate and interrupts
   settings_init(); // Load grbl settings from EEPROM
   st_init(); // Setup stepper pins and interrupt timers
-  sei(); // Enable interrupts
+  //sei(); // Enable interrupts
   
   memset(&sys, 0, sizeof(sys));  // Clear all system variables
   sys.abort = true;   // Set abort to complete initialization
